docs/quick_start.md

```diff
@@ -0,0 +1,46 @@
+# Quick start
+
+## Getting it
+Download it from repo.spring.io or Maven Central repositories (stable releases only):
```

Comment:
I'd revert the order the repos are mentioned here, as the snippet and badge below show a SNAPSHOT.